### DISCUSSION TITLE
Where do generic category names show up in the UI?

### DISCUSSION THREAD

**Discussion by RicoElectrico** [organicmaps/organicmaps#5098] [Translations]
I am in the process of reviewing, fixing and completing the Polish translation. I am confused about strings like `[type.man_made]` or `[type.landuse]` and so on. Are they ever displayed in the app? The particular types like `[type.man_made.chimney]` or `[type.landuse.forest]` sure are, but I can't find the former.

**Comment by biodranik** [comment]
OpenStreetMap.org has many man_made tags. Those of them that are not explicitly supported by Organic Maps go into a generic man_made type. Try to find some objects using overpass turbo and select them.

**Reply by RicoElectrico** [reply]
This one for example has a `man_made=wastewater_plant` tag, but I can't select or find it:
https://www.openstreetmap.org/way/123267819
